mandatory/2-function-creation.js

```diff
@@ -25,7 +41,14 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+let arr = arrayOfStrings.length;
+
+for (let i = 0; i < arr.length; i++){
+
+return arr[i].trim() && arr.replaceAll("/", "") && arr.toLowerCase();
```

Comment:
```suggestion
return arr[i].trim().replaceAll("/", "").toLowerCase();
```
&& is a logical operator, it doesn't join the array methods together. All you should do is list them in the order you want them to be done with a . before each one